parsec/core/invite/greeter.py

```diff
@@ -358,10 +358,8 @@ class DeviceGreetInProgress4Ctx:
     _shared_secret_key: SecretKey
     _cmds: BackendInvitedCmds
 
-    async def do_create_new_device(
-        self, author: LocalDevice, device_name: DeviceName, device_label: Optional[str]
-    ) -> None:
-        device_id = DeviceID(f"{author.user_id}@{device_name}")
+    async def do_create_new_device(self, author: LocalDevice, device_label: Optional[str]) -> None:
+        device_id = DeviceID(f"{author.user_id}@{DeviceName.new()}")
```

Comment:
A `DeviceID.new_from_user_id()` method might make sense?

Reply:
You touch something interesting: the DeviceID(f"{xxx.user_id}@whatever")` pattern is used multiple times throughout the codebase, I wonder if it we could provide a method to `UserID` instead to further simplify the use:
```python
author.user_id.to_device_id(DeviceName.new())
```
(not sure about the naming though...)

Reply:
> `author.user_id.to_device_id(DeviceName.new())`

Looks good! Could even be `author.user_id.new_device_id()`

Reply:
`author.user_id.to_device_id(DeviceName.new())` is more useful than `author.user_id.new_device_id()` given it allows to do `author.user_id.to_device_id("laptop")` which is used pretty often in the codebase